utils/ICutils.py

```diff
@@ -12,7 +13,7 @@
 import skimage
 
 # explicit import to PyBaSiC due to not having package support
-sys.path.append("./PyBaSiC/")
+sys.path.append("../1.preprocessing_data/PyBaSiC/")
```

Comment:
Double checking: is PyBasic still needed here?

Reply:
Same as above comment, I will be move ICutils into a ignored/hidden folder since we do not use it anymore.